packages/markdown/remark/src/shiki.ts

```diff
@@ -45,11 +45,18 @@ export async function createShikiHighlighter({
 	defaultColor,
 	wrap = false,
 	transformers = [],
+	langAlias = {},
 }: ShikiConfig = {}): Promise<ShikiHighlighter> {
 	theme = theme === 'css-variables' ? cssVariablesTheme() : theme;
+	const aliasKeys = [];
 
+	for (const [fromLang, toLang] of Object.entries(langAlias)) {
+		langs?.push(toLang);
+		aliasKeys.push(fromLang);
+	}
 	const highlighter = await getHighlighter({
 		langs: ['plaintext', ...langs],
+		langAlias,
```

Comment:
I think it's probably better to pass `langAlias` directly instead of adding it to the `langs` automatically so that we don't modify the behaviour of shiki's native options. What do you think?

Also `aliasKeys` seem unused.

Reply:
> I think it's probably better to pass langAlias directly instead of adding it to the langs automatically so that we don't modify the behaviour of shiki's native options. What do you think?

Unfortunately, it doesn't work. The tests fail: 

```
  Error [ShikiError]: Failed to parse Markdown file "undefined":
  Language `cjs` not found, you may need to load it first 
```

This happens because the list `langs` is empty, so there's not language to map from `cjs` to `javascript`

Reply:
OK, I see what you mean @bluwy 

I updated the code and updated the changeset too. Let me know what you think